products/nativescript/tns.py

```diff
@@ -362,3 +363,15 @@ def kill():
         else:
             Process.kill(proc_name='node', proc_cmdline=Settings.Executables.TNS)
             Process.kill_by_commandline(cmdline='webpack.js')
+
+    @staticmethod
+    def preview(app_name, bundle=False, hmr=False, aot=False, uglify=False, wait=False,
```

Comment:
If tns preview command have no `--justlaunch` option may be always that to use `wait=False`, if so may be we can remove it as param (since it is always False).

Reply:
:+1:

Reply:
Actually in Tns.exec_command method  which we use here,  wait by default is True, so we heed to specify for preview it needs to be False.